src/pudl/metadata/fields.py

```diff
@@ -4811,6 +4904,10 @@
         "type": "date",
         "description": "The record in the changelog is valid until this date. The record is valid from the report_date up until but not including the valid_until_date.",
     },
+    "value": {
+        "type": "string",
+        "description": "String value of data point.",
+    },
```

Comment:
This seems very vague without more context. What does it mean?

Reply:
Same as above, I've changed the naming for this table to hopefully make it more clear.